hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/curator/ZKCuratorManager.java

```diff
@@ -34,31 +39,32 @@
 import org.apache.hadoop.fs.CommonConfigurationKeys;
 import org.apache.hadoop.security.SecurityUtil;
 import org.apache.hadoop.security.authentication.util.JaasConfiguration;
+import org.apache.hadoop.util.Preconditions;
 import org.apache.hadoop.util.ZKUtil;
 import org.apache.zookeeper.CreateMode;
 import org.apache.zookeeper.Watcher;
 import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.client.ZKClientConfig;
+import org.apache.zookeeper.common.ClientX509Util;
 import org.apache.zookeeper.data.ACL;
 import org.apache.zookeeper.data.Stat;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import org.apache.hadoop.util.Preconditions;
 
 /**
  * Helper class that provides utility methods specific to ZK operations.
  */
 @InterfaceAudience.Private
 public final class ZKCuratorManager {
 
-  private static final Logger LOG =
-      LoggerFactory.getLogger(ZKCuratorManager.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ZKCuratorManager.class);
```

Comment:
I like cleaning this kind of things but it adds too much churn for a change that should be more contained.
I would prefer having a PR just for cleaning up the code than adding some feature while fixing other things in the side.
Let's avoid in this PR:
* Changing import.
* Adjusting line length breaks.
* Adjusting comments.
* Changing order.

Happy to review the followup for those.

Reply:
Thank you for your feedback!